src/Storageless/Http/SessionMiddleware.php

```diff
@@ -248,25 +248,21 @@ public function extractSessionContainer(Token $token = null) : SessionInterface
     private function appendToken(SessionInterface $sessionContainer, Response $response, Token $token = null) : Response
     {
         $sessionContainerChanged = $sessionContainer->hasChanged();
-        $sessionContainerEmpty   = $sessionContainer->isEmpty();
```

Comment:
This was cached on purpose to avoid calling `isEmpty()` twice, since the sessiondata is potentially mutable: can it be restored, or is this removal part of tge fix?